frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/sidebar.dart

```diff
@@ -94,37 +94,43 @@ class HomeSideBar extends StatelessWidget {
           right: BorderSide(color: Theme.of(context).dividerColor),
         ),
       ),
-      child: Padding(
-        padding: const EdgeInsets.symmetric(horizontal: 12),
-        child: Column(
-          mainAxisSize: MainAxisSize.min,
-          mainAxisAlignment: MainAxisAlignment.start,
-          children: [
-            // top menu
-            const SidebarTopMenu(),
-            // user, setting
-            SidebarUser(user: user, views: views),
-            const VSpace(20),
-            // scrollable document list
-            Expanded(
+      child: Column(
+        mainAxisSize: MainAxisSize.min,
+        mainAxisAlignment: MainAxisAlignment.start,
+        children: [
+          // top menu
+          const Padding(
+            padding: EdgeInsets.symmetric(horizontal: 12),
+            child: SidebarTopMenu(),
+          ),
+          // user, setting
+          Padding(
+            padding: const EdgeInsets.symmetric(horizontal: 12),
+            child: SidebarUser(user: user, views: views),
+          ),
+          const VSpace(20),
+          // scrollable document list
+          Expanded(
+            child: Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 12),
```

Comment:
You can extract this padding to a const:
```
const _menuHorizontalInset = EdgeInsets.symmetric(horizontal: 12);
```

Reply:
Sure! Will make the mentioned changes